dev/breeze/src/airflow_breeze/utils.py

```diff
@@ -0,0 +1,69 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import hashlib
+import shlex
+import subprocess
+
+# from subprocess import PIPE, Popen, STDOUT
+from typing import Dict, List
+
+# def log_subprocess_output(pipe):
+#     for line in iter(pipe.readline, b''): # b'\n'-separated lines
+#         print('got line from subprocess: %r', line)
+
+
+# def new_run_command(cmd: List[str], *, check: bool = True, **kwargs):
+#     print(f"$ {' '.join(shlex.quote(c) for c in cmd)}")
+#     try:
+#         process = Popen(cmd, stdout=PIPE, stderr=STDOUT)
+#         with process.stdout:
+#             log_subprocess_output(process.stdout)
+#         exitcode = process.wait() # 0 means success
+#     except subprocess.CalledProcessError as ex:
+#         print("========================= OUTPUT start ============================")
+#         print(ex.stderr)
+#         print(ex.stdout)
+#         print("========================= OUTPUT end ============================")
+#         raise
+
+
+def run_command(cmd: List[str], *, check: bool = True, **kwargs):
+    print(f"$ {' '.join(shlex.quote(c) for c in cmd)}")
```

Comment:
This line should only be printed in verbose mode - and likely in "[blue]".

Reply:
ok @potiuk